.github/workflows/integrationTests.yaml

```diff
@@ -6,6 +6,7 @@ on:
       - main
       - 3.x
       - 4.x
+      - 5.x
```

Comment:
we may need to also add 5.x to .github/workflows/NativeTests.yaml

Reply:
Done